packages/apollos-data-prayer/src/prayer-requests/schema.js

```diff
@@ -3,18 +3,18 @@ import gql from 'graphql-tag';
 const prayerRequestSchema = gql`
   extend type Query {
     getPublicPrayerRequests: [PrayerRequest]
-    getPublicPrayerRequestsByCampus(campusId: Int!): [PrayerRequest]
+    getPublicPrayerRequestsByCampus(campusId: String!): [PrayerRequest]
     getCurrentPersonPrayerRequests: [PrayerRequest] @cacheControl(maxAge: 0)
     getPrayerRequestsByGroups: [PrayerRequest]
   }
   extend type Mutation {
     addPublicPrayerRequest(
-      CampusId: Int!
+      CampusId: String!
       CategoryId: Int!
       Text: String!
       FirstName: String!
       LastName: String
-      IsAnonymous: String
+      IsAnonymous: Boolean
```

Comment:
If I recall from my work on this, the `IsAnonymous` is a dumb property that stores 'True' and 'False' as strings and not booleans. What was the reason for reverting this to a boolean?

Reply:
because a bit is smaller than a five-byte character ;)

Reply:
bools should *always* be used when they can be. It makes more sense to a developer trying to write client code too.

Reply:
I don't disagree, I'm just telling you how this is handle on Rock's end which cannot be changed. My guess is that the IsAnonymous functionality is now broken.

Reply:
Nah dawg, I fixed that in the data source function.

Reply:
Sweet.